plugins/lookup/github_app_access_token.py

```diff
@@ -47,6 +47,14 @@
       - How long the token should last for in seconds.
     default: 600
     type: int
+  github_api_url:
+    description:
+      - Base URL for the GitHub API (for GitHub Enterprise Server).
+      - If not provided, defaults to C(https://api.github.com).
+      - Example: C(https://github-enterprise-server.example.com/api/v3)
```

Comment:
The current error in CI is because of this line, it should be:
```suggestion
      - "Example: C(https://github-enterprise-server.example.com/api/v3)"
```
Without the quotes, this list entry is a YAML dictionary with key `Example` and value `C(https://github-enterprise-server.example.com/api/v3)`.